docs/translations.md

```diff
@@ -1,3 +1,53 @@
+# Managing strings
+
+We use Android's [resource system](https://developer.android.com/guide/topics/resources/localization) to localize
+user-visible strings in our apps.
+
+Our source language is English (American English to be more precise, but simply "English" (en) on Weblate).
+
+Feature modules are expected to store strings exclusively used by them in their `res/values/strings.xml` file.
+There's currently no final place for strings shared between feature modules. For now there's some in
+[:legacy:ui:base](/legacy/ui/base/src/main/res/values/strings.xml).
+
+Translations of source strings happen exclusively in our
+[Weblate project](https://hosted.weblate.org/projects/tb-android/). This means the source language is only modified by
+changes to this repository, i.e. via pull requests. Translations are only updated on Weblate and then merged into this
+repository by the Thunderbird team. This is to avoid overlapping changes in both repositories that will lead to merge
+conflicts.
+
+## Adding a string
+
+Add a new string to the appropriate `res/values/strings.xml` file. Be careful when using Android Studio's translations
+editor. It might modify files containing translations. If so, revert those changes before committing the changes.
+
+## Changing a string
+
+Changing a string should be avoided. Weblate doesn't automatically invalidate translations when a source string is
+changed. This can be worked around by removing the old string and adding a new one. Make sure to only modify the source
+language. It's fine for the translations to then contain unused strings. The next merge with Weblate will remove those.
+
+## Removing a string
+
+Remove the source string from `res/values/strings.xml`. Don't modify translations under `res/values-<lang>/strings.xml`.
+The next merge from Weblate will automatically get rid of the translated strings.
+
+## Changing translations in this repository
+
+This should be avoided whenever possible! It's also something that should only be done by team members that have the
+necessary permission to lock translations on Weblate.
+
+So if you absolutely have to change translated strings in this repository, follow this procedure:
```

Comment:
Slightly more positive framing:
```suggestion
This should be avoided whenever possible, as it can create merge conflicts between Weblate and this repository. If you need to change individual strings, please translate them on Weblate instead. 

If a mechanical change is necessary across all languages that doesn't require langauge translation, this should be discussed with the core team who will use this procedure:
```